lib/home.dart

```diff
@@ -17,12 +27,23 @@ class Home extends StatelessWidget {
             SizedBox(
               width: 10,
             ),
-            Text('Sound Doctrine', style: Theme.of(context).textTheme.title),
+            Text('Sound Doctrine'),
           ],
         ),
       ),
       body: NavigationPage(),
       drawer: NavigationDrawer(),
+      floatingActionButton: FloatingActionButton(
```

Comment:
I think utilising the pre-existing nav bar to toggle dark on/off would be preferable unfortunately, as it seems to get in the way of viewing!